### ISSUE TITLE
Unhide Meetings! This is important missing!

### ISSUE THREAD

**Issue by CoderA07** [TobiasPankner/Teams-Auto-Joiner#86]
Hi TobiasPankner.

I'm live on Turkey Country Hearing impaired (Deaf) (Not a problem. I'm talking on sound. :) )

The live captions only english work. My Teachers talks Turkish Language. Not Problem my me. I solved problem for He is recorded video to subtitles my artifical intellignce caption software 👍 

Thank you big much helping automatic joining teams meeting : )

Aand the is big big.. Problem to..
This is important missing feature in unhiding meeting function!
Yes. The Teachers opened Channels to meeting subjects he week.

Example:

TALKING TEAM (Main Channel Ok.)
-General (Working.)
-1.week HELLO EVERYONE (Working. I unhiddened channel)
-2.week HOW TO MAKE (Working. I unhiddened channel)
( 1 Hidden channel ) (Not working! This is big problem! I fixing manually unhiding channel. )

Please add for automatic unhide subjects  that is, there is the subject meeting.

Because sometimes I have a hard time looking at this. The meeting is mandatory for me. So he's taking attendance.

Thank you.

**Comment by CoderA07** [comment]
@TobiasPankner 
It upset me not to answer us while answering other issues :(

**Comment by TobiasPankner** [comment]
So the channel is hidden by default when it is created?

**Comment by CoderA07** [comment]
Yes. As soon as the channel is opened, the teachers make themselves hide.
